content/projects/katas/level-1/_index.md

```diff
@@ -177,4 +177,47 @@ eg `combine([11,22,33], [1,2,3])` should return
 
 ```
 [11,1,22,2,33,3]
-```
\ No newline at end of file
+```
+
+## Instructions for Reviewers
```

Comment:
This is a bit too hi-res and you are repeating many of the instructions that are listed above, lets get this cleaned up :)